src/Pages/addPhoto/AddPhoto.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import './AddPhoto';
+import Header from '../../Components/Header/header';
+import './AddPhoto.css';
+import { storage , store , timestamp  } from '../../Components/firebase';
+import LoaderModel from '../../Components/Loader/LoaderModal';
+import Button from '@material-ui/core/Button';
+import AddIcon from '@material-ui/icons/AddAPhoto';
+import DeleteIcon from '@material-ui/icons/DeleteForeverRounded';
+const types = ['image/png' , 'image/jpeg'];
+
+
+class AddPhoto extends React.Component {
+    constructor(props) {
+        super(props);
+         this.state = { 
+             pictures: [] ,
+             image : '',
+             url : '',
+             imageId:null,
+             isUpload:null,
+             isDelete:null
+        };
+    }
+
+
+      uploadImg = (e) => {
+        
+        const selected = e.target.files[0];
+        if(e.target.files[0] && types.includes(selected.type)){
+            this.setState({url:'',isUpload:1});
+            var url='';
+            const image = e.target.files[0];
+
+            const uploadTask = storage.ref(`images/${image.name}`).put(image);
+            const collectionRef = store.collection('images');
+            uploadTask.on('state_changed',
+            (snapshot) => {
+                // progress function
+            },
+            (error) => {
+                // error function
+                console.log(error);
+            },
+            async () => {
+                // complete function
+                await storage.ref('images').child(image.name).getDownloadURL()
+                .then((imgUrl) => {
+                    console.log(imgUrl);
```

Comment:
this is not how you use async-await. it is supposed to replace the use of .then(). You are mixing both of them here.